tasks/interview-basic-coreJS.md

```diff
@@ -0,0 +1,73 @@
+## Question examples:
+
+### JavaScript:
+
+- #### Basics
+
+  - Data types
+  - Number methods
+  - String methods
+  - RegExp, quantifiers
```

Comment:
let var const - differences
what is 'use strict' 
ternary operator
switch case - examples, where it can be useful
arrow func/ func expression/ func declaration
what is polyfills 
Date: object, methods
async/defer scripts 
type conversions